python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -958,6 +959,160 @@ def corr(self, method="pearson", min_periods=1):
 
         return res
 
+    def cov(self, min_periods=0, ddof=1):
+        """
+        Compute the pairwise covariance among the columns of a DataFrame,
+        excluding NA/null values.
+
+        The returned data frame is the covariance matrix of the columns of
+        the DataFrame.
+
+        Both NA and null values are automatically excluded from the
+        calculation.(See the note below about bias from missing values)
+        A threshold can be set for the minimum number of observations
+        for each value created.Comparisons with observations below this
+        threshold will be returned as NaN.
+
+        This method is generally used for the analysis of time series data to
+        understand the relationship between different measures across time.
+
+        Parameters
+        ----------
+        min_periods: int, optional
+            Minimum number of observations required per pair of columns
+            to have a valid result.
+
+        ddof: int, optional
+            Delta degrees of freedom, default is 1.
+
+        Returns
+        -------
+        DataFrame
+            Covariance matrix.
+
+        Notes
+        -----
+        Returns the covariance matrix of the DataFrame’s time series.
```

Comment:
Use straight quotes.
```suggestion
        Returns the covariance matrix of the DataFrame's time series.
```